### PR TITLE
Move more information to wiki and add build instructions

### LINKED ISSUE #11: Move data from README.md to wiki
This started out as non-public / non-github.
Now that the project has a github wiki, the information should be moved to the wiki

(I've already moved the "Soundtrack" section to the wiki [PR to remove it from readme welcome])

### PR DESCRIPTION
**Pull request by JayFoxRox**
Fixes #11, #5 and #7

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,159 +17,28 @@ Additionally the file `data/model/Daylight Box UV.png` is by [KIIRA](https://ope
...
-- https://soundcloud.com/peterpanspeedrock
-- https://soundcloud.com/thevines/02-ladybug << ?
-- https://www.youtube.com/watch?v=adnAIJUnyYg << ?????
-- https://www.youtube.com/watch?v=bIaQ9VsnHnU
-- https://www.youtube.com/watch?v=CwFmidRtQcw << Whole channel is awesome it seems!
-- https://www.youtube.com/watch?v=JmBbF8JXRBc
-- https://www.youtube.com/watch?v=-QA0WLSZqJk << Potentially, easy to get permission [hopefully :P]
-- https://www.youtube.com/watch?v=TqwZDZexRg0
-
+Now you can run the generated executable called `main`
```
Comment (tulior) [inline README.md]: `./Boat-Racing-Game/build $ ./main`
On my end, I couldn't run the `main` executable directly from the `build` folder. It probably has something to do with some in-game path's reference?
```
./Boat-Racing-Game/build $ cd ..
./Boat-Racing-Game $ ./build/main
```
The above works.

Comment (JayFoxRox) [reply README.md]: Thanks!

I'll try to find a better solution. I guess we should have cmake copy data files to the build folder automatically (but only the final (*.obj, *.png, ..) files, not project (*.blend, ..) files)

**Comment by anodium** [comment]
Regarding #5 and #11, well...

```
Please help me finalize the wiki articles so they can be moved into the source tree.

Dependency on github wikis should be avoided.
```

From the [repo's GitHub wiki](https://github.com/JayFoxRox/Boat-Racing-Game/wiki).

**Comment by JayFoxRox** [comment]
@anodium the idea is:

- Clean up the README asap so people are not hit by a wall of text (in a similar fashion I intend to move the source into subdirectories) => Move README content to wiki articles
- Continue work on the wiki articles (temporary dependency)
- Once the wiki articles are finalized [= not changing much anymore] they can be moved back into the source tree (but not README.md)

(Some wiki articles are very WIP stuff too which will be turned into code or data files at some point)

**Comment by anodium** [comment]
@JayFoxRox Then why not move them into `docs/` within the repository, or something of the sort?

**Comment by JayFoxRox** [comment]
That's planned once those articles are finalized, but I think keeping such docs on the wiki for now increases productivity + creativity. It also avoid a lengthy review for each change.

**Comment by anodium** [comment]
Fair enough ^.^

Maybe the GitHub wiki could be submodule'd into the main repository for now, then, when done, the two trees can be merged together like [this](http://stackoverflow.com/a/1789374/7650483) (second method), thus preserving the history of the wiki as well.

**Comment by JayFoxRox** [comment]
~~Depends on #13 now (so the build instructions work without manually moving data files)~~

Will merge this tomorrow probably if no further review happens

**Pull request merged**